dev-python/gitdb2/gitdb2-2.0.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="GitDB is a pure-Python git object database"
+HOMEPAGE="
+	https://github.com/gitpython-developers/gitdb
+	https://pypi.python.org/pypi/gitdb2"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	!dev-python/gitdb[${PYTHON_USEDEP}]
+	>=dev-python/smmap2-2.0.1[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}"
+# Testsuite require files from a git repo
```

Comment:
Tests from the github tarball fail with:
```
-------------------- >> begin captured logging << --------------------
root: INFO: You can set the GITDB_TEST_GIT_REPO_BASE environment variable to a .git repository of your choice - defaulting to the gitdb repository
```

You really need a repo, not just a snapshot/release.